mdsuite/calculators/green_kubo_ionic_conductivity.py

```diff
@@ -230,39 +223,80 @@ def _post_operation_processes(self):
         -------
 
         """
-        result = self.prefactor * np.array(self.sigma)
-
+        self.sigmas = np.array(self.sigmas)
```

Comment:
It should work without this line I think. `np.mean` should convert it to a `np.ndarray`